src/System.ComponentModel.Primitives/src/System/ComponentModel/Component.cs

```diff
@@ -120,9 +120,9 @@ protected virtual void Dispose(bool disposing)
         /// <see cref='System.ComponentModel.Component'/>, if any.
         /// This method should not be overridden.
         /// </summary>
-        public override string ToString()
+        public override string? ToString()
         {
-            ISite s = _site;
+            ISite? s = _site;
```

Comment:
I don't think ToString could be returning null here. You should be able to change return type back to string

Reply:
I agree, just because of Type.FullName return nullable, not sure if i should suppress that and make it non null

Reply:
GetType().FullName can definitely return null if the type has GenericTypeParameters: https://source.dot.net/#System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs,1491 -- however since this type is not generic it will never be null, so I think we should just use `!` on the return.

Reply:
Will update thanks!